### PR TITLE
chore(broker): Keep records of active long polling requests

### LINKED ISSUE #4821: Backport PR4681 to 0.22.x
**Description**

The bug fix #4681 needs to be back ported to 0.22.x

### PR DESCRIPTION
**Pull request by pihme**
## Description

Backport of PR4681 to 0.22

## Related issues

closes #4821

## Pull Request Checklist

- [X] All commit messages match our [commit message guidelines](https://github.com/zeebe-io/zeebe/blob/develop/CONTRIBUTING.md#commit-message-guidelines)
- [X] The submitting code follows our [code style](https://github.com/zeebe-io/zeebe/wiki/Code-Style)
- [X] If submitting code, please run `mvn clean install -DskipTests` locally before committing

### CODE REVIEW

File: parent/pom.xml [hunk 1]
```diff
@@ -44,7 +46,7 @@
     <version.commons-logging>1.2</version.commons-logging>
     <version.commons-math>3.6.1</version.commons-math>
     <version.commons-codec>1.14</version.commons-codec>
-    <version.elasticsearch>6.8.6</version.elasticsearch>
+    <version.elasticsearch>6.8.9</version.elasticsearch>
```
Comment (deepthidevaki) [inline parent/pom.xml:49]: Is this required? I think it is good not to change the versions when backporting.

**Review by deepthidevaki — CHANGES_REQUESTED** [review]
LGTM. Just one minor comment.

File: parent/pom.xml (hunk 1 above)
Comment (pihme) [reply parent/pom.xml:49]: Required - yes and no

There was a security issue with the log4j library. On bumping the log4j version, then the Elasticsearch version had a dependency version conflict, so I bumped it, too.

I agree with your sentiment that changing versions during backport is not trivial. My personal rule of thumb is that I try to limit it to patch version changes only.

If you insist I can roll it back. In this case I would open another issue to bump the log4j version.

Please advise.

**Review by deepthidevaki — APPROVED** [review]

**Comment by npepinpe** [comment]
Yes, it's confused because it's looking for POMs that didn't exist yet in 0.22. Side note, I thought I sent you an invite for snyk, didn't you receive it?

**Comment by pihme** [comment]
@npepinpe I did get the invite and I could see the errors.

**Pull request merged**